.spectral.yaml

```diff
@@ -102,3 +102,11 @@ rules:
       function: pattern
       functionOptions:
         notMatch: 'string'
+
+  schema-name-length:
+    description: "Schema name must not be too long to support client generation. Current limitation come from the Ruby using tar which have a limit of 100 characters."
+    given: $.components.schemas[*]~
+    then:
+      function: length
+      functionOptions:
+        max: 95
```

Comment:
why 95 over 99 or 100?

Reply:
Real limit is 100, minus the extensino + small buffer just to be safe.